DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py

```diff
@@ -102,9 +107,22 @@
 if (process.runType.getRunType() == process.runType.hi_run):    
     process.load('Configuration.StandardSequences.ReconstructionHeavyIons_cff')
     process.load('Configuration.StandardSequences.RawToDigi_Repacked_cff')
-    process.siPixelDigis.InputLabel   = cms.InputTag("rawDataRepacker")
-
-
+    process.castorDigis.InputLabel = cms.InputTag("rawDataRepacker")
+    process.csctfDigis.producer = cms.InputTag("rawDataRepacker")
+    process.dttfDigis.DTTF_FED_Source = cms.InputTag("rawDataRepacker")
+    process.ecalDigis.InputLabel = cms.InputTag("rawDataRepacker")
+    process.ecalPreshowerDigis.sourceTag = cms.InputTag("rawDataRepacker")
+    process.gctDigis.inputLabel = cms.InputTag("rawDataRepacker")
+    process.gtDigis.DaqGtInputTag = cms.InputTag("rawDataRepacker")
+    process.gtEvmDigis.EvmGtInputTag = cms.InputTag("rawDataRepacker")
```

Comment:
hi @fioriNTU - this line is presumably the cause of your problem... you might consider using this helper function to help with future maintenance. 

https://github.com/cms-sw/cmssw/blob/master/FWCore/ParameterSet/python/MassReplace.py#L119

Reply:
@davidlange6 even commenting that line I get the same crash. Thanks for the information about the helper function.